fcrepo-http-api/src/main/java/org/fcrepo/http/api/ContentExposingResource.java

```diff
@@ -214,11 +219,58 @@ protected Response getContent(final String rangeValue,
         return ok(outputStream).build();
     }
 
+
+    protected URI getURLAccessType(final MediaType mediaType) {
+        return URI.create(getAccessTypeValue(mediaType.getParameters(), URL_ACCESS_TYPE));
+    }
+
+    /**
+     * Throws a NotSupportedException if media type is an invalid message/external-body type. To be valid it must
+     * contain the access-type parameter that equals a valid access type and a URL parameter that contains a URL.
+     * 
+     * @param mediaType The media type to be validated
+     * @throws NotSupportedException
+     */
+    protected void checkExternalBody(final MediaType mediaType) {
+        if (isExternalBody(mediaType)) {
```

Comment:
All uses of this function seem to have been  in a `if (isExternalBody(foo)) {` already, so this seems to be a redundant check.

Reply:
Or perhaps `isExternalBody()` and `checkExternalBody()` should be one combined function?